automx2/config.py

```diff
@@ -30,6 +30,9 @@
 CONF_DB_URI = 'db_uri'
 CONF_LOGLEVEL = 'loglevel'
 CONF_PROXY_COUNT = 'proxy_count'
+SERVER_CERT = 'server_cert'
```

Comment:
Keep the configuration style `CONF_SERVER_CERT` etc., Or even because this applies only to mobileconfig `CONF_MOBILECONFIG_SIGN_CERT` `CONF_MOBILECONFIG_SIGN_KEY` `CONF_MOBILECONFIG_SIGN_CA`